src/blocks/fileUpload/FileUpload.tsx

```diff
@@ -0,0 +1,49 @@
+import { DragEventHandler, FC, forwardRef, ReactNode } from 'react';
+import styled, { FlattenSimpleInterpolation, css } from 'styled-components';
+
+export type FileUploadProps = {
+  children?: ReactNode;
+  css?: FlattenSimpleInterpolation;
+  disabled?: boolean;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  onDrop?: DragEventHandler;
+  hidden?: boolean;
+  id: string;
```

Comment:
Don't think we need these two props

Reply:
input will be hidden all the time

Reply:
We need id because we are adding a label in the children . Because with this id only it opens the upload option.

<label htmlFor="file-upload">Browse to Choose/Upload </label>

Reply:
I will remove the hidden tag and make it true for all the case